Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -140,31 +132,38 @@ public class CameraManager {
             return
         }
 
+        internalAnimator?.stopAnimation()
```

Comment:
what should happen if the animator's completion block has already been scheduled?

Reply:
this is fine - it doesn't cause the completion block to be invoked a second time and we _do_ want the initial completion block to be invoked.